src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
-import * as Events from './events';
+import * as Dom from './utils/dom.js';
+import * as Events from './utils/events.js';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'object.assign';
```

Comment:
Going to submit a PR to `object.assign` that will give us some way to grab the correct `assign` function.
Perhaps something like `import {assign} from 'object.assign'` as opposed to `import assign from 'object.assign'` but we won't need to do the above ourselves.